docs/requirements-docs.txt

```diff
@@ -1,10 +1,10 @@
+pygments>=2.7.4 # not directly required, pinned by Snyk to avoid a vulnerability
+recommonmark
+sphinx>=3.0.4 # not directly required, pinned by Snyk to avoid a vulnerability
+sphinx-copybutton
+sphinx-prompt 
 # Copyright (C) 2020-2024 Intel Corporation
```

Comment:
Copyright should appear at the beginning of the file

Reply:
resolved in commit 3e047fd060cb563fdaddf059b06af1df45ea6b37